smll/FaceDetector.cpp

```diff
@@ -293,7 +297,7 @@ namespace smll {
 
 		}
 		// If faces are not found
-		if (m_faces.length == 0 && !trackingFailed) {
+		if (m_faces.length == 0 && !trackingFailed && !wasFaceDetected) {
             // Wait for 5 frames and do face detection
```

Comment:
Why do we need !wasFaceDetected condition here?

Reply:
The condition is added, not to wait for 5 frames when a face was detected before and force redetect in that case.